### ISSUE TITLE
Settings editor indicators not accessible by keyboard

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#159088]
Currently, the Settings indicators area, which is the area with labels such as "Modified elsewhere", is not accessible by keyboard.

Ideas:
- [ ] Change the key event handling so that the user can tab once into the indicators container, and then use left/right arrows to focus on different indicators.
- [x] Change the key event handling so that once focus is on an indicator, the user can use space or enter keys, or Ctrl+K Ctrl+I to show and focus into the hover.
- [ ] Change the appearance of the indicators so that it is more obvious that one can tab to them. Right now, the indicators are shown in plain text, and have underlines when hovered over. Yet, users cannot click on them; they are only supposed to hover over them. Moreover, after enabling tabIndex for them and adding an outline to make it clear when they received focus, because they were plain text, the appearance just seemed off, as though the user was "magically" focusing onto plain text.
